orchagent/routeorch.cpp

```diff
@@ -1577,6 +1596,19 @@ bool RouteOrch::updateNextHopRoutes(const NextHopKey& nextHop, uint32_t& numRout
         return true;
     }
 
+    /* Check if nexthop is mux nexthop */
+    MuxOrch* mux_orch = gDirectory.get<MuxOrch*>();
+    NextHopGroupKey nhg_key;
+    if (inNextHopGroup(nextHop, nhg_key) && mux_orch->isMuxNexthops(nhg_key))
```

Comment:
What happens if a nexthop is part of NHG but also another route points to this single nexthop?
For e.g like NH1 below:

R1 -> NHG1 (NH1, NH2)
R2 -> NH1